src/explore-education-statistics-admin/src/pages/release/edit-release/data/ReleaseDataUploadsSection.tsx

```diff
@@ -265,73 +269,96 @@ const ReleaseDataUploadsSection = ({
               </>
             )}
 
-            {dataFiles.map(dataFile => (
-              <SummaryList
-                key={dataFile.filename}
-                additionalClassName="govuk-!-margin-bottom-9"
-              >
-                <SummaryListItem term="Subject title">
-                  <h4 className="govuk-heading-m">{dataFile.title}</h4>
-                </SummaryListItem>
-                <SummaryListItem term="Data file">
-                  <ButtonText
-                    onClick={() =>
-                      service
-                        .downloadDataFile(releaseId, dataFile.filename)
-                        .catch(handleApiErrors)
-                    }
-                  >
-                    {dataFile.filename}
-                  </ButtonText>
-                </SummaryListItem>
-                <SummaryListItem term="Metadata file">
-                  <ButtonText
-                    onClick={() =>
-                      service
-                        .downloadDataMetadataFile(
-                          releaseId,
-                          dataFile.metadataFilename,
-                        )
-                        .catch(handleApiErrors)
+            <Accordion id="data-files">
+              {dataFiles.map((dataFile, index) => (
+                <AccordionSection
+                  /* eslint-disable-next-line react/no-array-index-key */
+                  key={`${dataFile.title}-${index}`}
+                  headingId={`${dataFile.title}-${index}`}
+                  heading={`${dataFile.title}`}
+                  onToggle={() => {
+                    const accId = `${dataFile.title}-${index}`;
```

Comment:
Move the `accId` up into the above lambda so you can re-use it for `key`, `headingId` and `open`.